### PR TITLE
Upgrade to net6 and update package versions

### PR DESCRIPTION
**Pull request by sprucely**

### CODE REVIEW

File: Keas.Mvc/Startup.cs [hunk 1]
```diff
@@ -227,7 +225,6 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerF
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
-                app.UseDatabaseErrorPage();
```
Comment (jSylvestre) [inline Keas.Mvc/Startup.cs:230]: Deleted this on purpose?

Comment (jcubbage) [reply Keas.Mvc/Startup.cs:230]: Might have gotten warning on that line. It is obsolete and will be removed. Looks like there is a preferred method: https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.builder.databaseerrorpageextensions.usedatabaseerrorpage?view=aspnetcore-6.0

Comment (sprucely) [reply Keas.Mvc/Startup.cs:230]: Yeah, there's a new `services.AddDatabaseDeveloperPageExceptionFilter();` that adds a db exception handler to the developer exception page stuff.

**Review by jSylvestre — APPROVED** [review]

**Review by srkirkland — APPROVED** [review]
🚢

**Pull request merged**